src/main/java/mate/academy/springboot/datajpa/model/Category.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+public class Category {
```

Comment:
add also `@Table(name = "categories")`

What about equals and hashcode? Same for Product but do not forget to exclude lazy fields from it

Reply:
not fixed this part of the comment: What about equals and hashcode? Same for Product but do not forget to exclude lazy fields from it